Aquality.Selenium.Template/Aquality.Selenium.Template.NUnit/Steps/MainPageSteps.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Aquality.Selenium.Browsers;
+using Aquality.Selenium.Template.Constants;
+using Aquality.Selenium.Template.Forms.Pages;
+using Aquality.Selenium.Template.NUnit.Extensions;
+using NUnit.Framework;
+using System;
+
+namespace Aquality.Selenium.Template.NUnit.Steps
+{
+    public class MainPageSteps : BaseSteps
+    {
+        private readonly MainPage mainPage;
+
+        public MainPageSteps()
+        {
+            mainPage = new MainPage();
```

Comment:
why do we need the default constructor, can't we init field inline? Or is the DI container usage missed?